plugin/completion.py

```diff
@@ -181,22 +282,23 @@ def format_completion(self, item: dict, can_resolve_completion_items: bool) -> s
 
         lsp_label = item["label"]
         lsp_filter_text = item.get("filterText")
-        lsp_detail = item.get("detail") or ""
-        lsp_detail = html.escape(lsp_detail.replace('\n', ' '))
-
-        if can_resolve_completion_items or "documentation" in item:
-            doc_link = '<a href="subl:lsp_resolve_docs">Documentation</a>'
-        else:
-            doc_link = ''
+        lsp_detail = html.escape(item.get("detail", "").replace('\n', ' '))
 
         if lsp_filter_text and lsp_filter_text != lsp_label:
             st_trigger = lsp_filter_text
             st_annotation = lsp_label
-            st_details = '{} {}'.format(doc_link, lsp_detail) if doc_link else lsp_detail
         else:
             st_trigger = lsp_label
-            st_annotation = lsp_detail
-            st_details = doc_link
+            st_annotation = ""
+
+        st_details = ""
+        if can_resolve_completion_items or "documentation" in item:
+            args = {
+                "item": deep_escape(item)
+            }
+            st_details += "<a href='subl:lsp_resolve_docs {}'>Δ</a>".format(sublime.encode_value(args))
+
+        st_details += "<p> {}</p>".format(lsp_detail)
```

Comment:
If `st_details` is still an empty string, does the extra space get rendered or does it not get rendered due to the `<p>` tags? (I'm not good at html)

Reply:
I just put a space to space it from the link a bit. 
I should have probably put the space on a different place, for example after the closing a tag`"...</a> "`. :)

Reply:
This should likely be a conditional add, I hate to add "space" at the bottom of the popup if it is literally going to be blank.

```python
st_details += "<p> {}</p>".format(lsp_detail) if lsp_detail else ""
```